recipes-dom0/dom0-image-weston/files_rcar/meta-xt-images-extra/inc/dom0-core-image.inc

```diff
@@ -0,0 +1,21 @@
+#Add Xen to build
+IMAGE_INSTALL_append = " \
+    xen-base \
+    xen-flask \
+    guests \
+"
+
+populate_append() {
+	install -m 0644 ${DEPLOY_DIR_IMAGE}/xen-${MACHINE}.gz ${DEST}/xen.gz
+}
+
+IMAGE_INSTALL_append = " \
+    guest-images \
```

Comment:
simply removing guest-images results in:
ERROR: linux-renesas-4.9.0+gitAUTOINC+23445f9f6a-r1 do_package: QA Issue: linux-renesas: Files/directories were installed but not shipped in any package:
  /xen
  /xen/domu.dtb
Please set FILES such that these items are packaged. Alternatively if they are unneeded, avoid installing them or delete them within do_install.
linux-renesas: 2 installed and not shipped files. [installed-vs-shipped]
ERROR: linux-renesas-4.9.0+gitAUTOINC+23445f9f6a-r1 do_package: Fatal QA errors found, failing task.

so, this not only needs guest-images removal, but also some rework of guests.bb